README.md

```diff
@@ -31,6 +31,7 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
    - [.NET Core 3.1.23 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-3.1.23-windows-x64-installer) or a newer 3.1.x runtime. This is needed currently for the Settings application.
    - [.NET 6.0.3 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-6.0.3-windows-x64-installer) or a newer 6.0.x runtime. 
    - [Microsoft Edge WebView2 Runtime](https://go.microsoft.com/fwlink/p/?LinkId=2124703) bootstrapper. This will install the latest version. 
+   - [Windows App SDK Runtime 1.0.1](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads). This will install version 1.0.1 if this or newer version is not installed already.
```

Comment:
This is on client side. Correct?

Then we need an issue for docs.msft too.

Reply:
It'll be installed by the installer, ideally ;)

Reply:
Yes. But we have the same list there.

Reply:
Lets log the bug and mark it as pri-0 against this release.  I do not want this PR to be held up

Reply:
Sure. This should not be a blocker. It should be a reminder.

Reply:
I still have a PR for docs open, ready to be accepted

Reply:
Issue #6715 is assigned to 0.58. If you can confirm that this will be in .58, I can easily do this in my PR which is still open...

Reply:
I want to wrap up the doc PR today. It is on my list to do for today. Let’s have a new PR for this so we can better schedule stuff